src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewEngines/CompositeViewEngine.cs

```diff
@@ -77,6 +85,14 @@ public ViewEngineResult GetView(string executingFilePath, string viewPath, bool
                 throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(viewPath));
             }
 
+            if (ViewEngines.Count == 0)
+            {
+                throw new InvalidOperationException(Resources.FormatViewEnginesAreRequired(
+                    typeof(MvcViewOptions).FullName,
+                    nameof(MvcViewOptions.ViewEngines),
+                    typeof(IViewEngine).FullName));
+            }
```

Comment:
Your comment here seems to conflict w/ the one [there](https://github.com/aspnet/Mvc/pull/4928/files#r69055323). Let's resolve this tomorrow morning.